### ISSUE TITLE
Why is `msg.value` part of the tip?

### ISSUE THREAD

**Issue by NIC619** [blunt-instruments/MevWallet#3]
Why is `msg.value` part of the tip to `subsidize` while there is already a `tip` parameter?

https://github.com/blunt-instruments/MevWallet/blob/82a34d4deb595f587be9517614eb7c367c920121/contracts/MevWalletV1.sol#L210

Also `msg.value` can be used in user's own execution, won't passing `msg.value` into `subsidize()` as tip further complicates the 
`loot` calculation?

https://github.com/blunt-instruments/MevWallet/blob/82a34d4deb595f587be9517614eb7c367c920121/contracts/MevWalletV1.sol#L156-L160

**Comment by prestwich** [comment]
So the purpose is to allow the user to send eth they do not own at the start of the transaction. Basically this creates an implied trade of the user's MEV and/or pre-existing MevWeth balance for the Searcher's ether input. This allows the user to call functions that require ether input _without having that ether_, then backfilling the ether via the searcher interaction. This means the user can make 1 transaction instead of 2 for common trades/collateralizations/etc that accept ether

Basically we do this to give MevWallet another super power ;)

You can see value checking logic here:

https://github.com/blunt-instruments/MevWallet/blob/82a34d4deb595f587be9517614eb7c367c920121/contracts/MevWalletV1.sol#L143-L161

**Comment by NIC619** [comment]
Oh that's right. I totally missed that caller is the searcher and `msg.value` is provided by the searcher. Now it all makes sense.

**Comment by NIC619** [comment]
Thanks for the prompt and helpful reply!
